File: PlayWall/src/main/java/de/tobias/playpad/action/actions/cart/handler/CartActionHandler.java
package de.tobias.playpad.action.actions.cart.handler;

import de.thecodelabs.midi.event.KeyEventType;
import de.tobias.playpad.action.actions.CartAction;
import de.tobias.playpad.pad.Pad;

public interface CartActionHandler
{
	void performAction(KeyEventType keyEventType, CartAction cartAction, Pad pad);
}

File: PlayWall/src/main/scala/de/tobias/playpad/action/actions/cart/handler/CartActionHandler.scala
package de.tobias.playpad.action.actions.cart.handler

import de.thecodelabs.midi.event.KeyEventType
import de.tobias.playpad.action.actions.CartAction
import de.tobias.playpad.pad.Pad

trait CartActionHandler {

	def performAction(keyEventType: KeyEventType, cartAction: CartAction, pad: Pad): Unit
}
